pkg/operator/ceph/nfs/spec.go

```diff
@@ -165,6 +166,7 @@ func (c *CephNFSController) daemonContainer(n cephv1.CephNFS, name string, binar
 		Args: []string{
 			"--", path.Join(k8sutil.BinariesMountPath, "rook"),
 			"ceph",
+			"--ceph-version-name", c.cephVersion.Name,
```

Comment:
Yes. I think this line needs to be removed. The init() routine in cmd/rook/ceph/nfs.go calls `addCephFlags(ganeshaConfigCmd)`, but does not do the same for ganeshaRunCmd. Thus, the `rook nfs run` command does not recognize the flag.

An alternate fix might be to add a call to `addCephFlags(ganeshaRunCmd)` in nfs.go. I'm unclear on which would be preferred.